test/test_container.py

```diff
@@ -118,6 +120,11 @@ def logger():
         yield patched
 
 
+@pytest.fixture(autouse=True)
```

Comment:
my comment wasn't so much about this test, as about our desired behaviour in general (e.g. when using `run`. we recently started adding deprecation warnings but users might not be seeing them